lib/machinery_helper.rb

```diff
@@ -34,17 +31,25 @@ def initialize(s)
     @system = s
     @arch = @system.arch
 
-    @local_helpers_path = "/usr/share/machinery/helpers"
+    @local_helpers_path = File.join(Machinery::ROOT, "machinery-helper")
+    @remote_helpers_path = File.join(Machinery::REMOTE_HELPERS_PATH, "machinery-helper")
   end
 
   def local_helper_path
-    File.join(@local_helpers_path, @arch, "machinery-helper")
```

Comment:
I wonder if it makes sense to keep the arch in the path, even though we only use one for now. That would make it clearer that it's architecture dependent and would also be more future-proof.